src/main/java/stsjorbsmod/powers/MirroredTechniquePower.java

```diff
@@ -0,0 +1,91 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardQueueItem;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import stsjorbsmod.util.CardMetaUtils;
+
+import java.lang.reflect.Field;
+
+public class MirroredTechniquePower extends CustomJorbsModPower {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(MirroredTechniquePower.class);
+    public static final String POWER_ID = STATIC.ID;
+
+    private static Field multiIntentField;
+    private static int extraPlays;
+
+    public MirroredTechniquePower(final AbstractCreature owner, final int extraPlays) {
+        super(STATIC);
+
+        this.owner = owner;
+        this.isTurnBased = false;
+        this.extraPlays = extraPlays;
```

Comment:
I think we'll want to consider the desired stacking behavior here; depending on design feedback, we might want to use amount for this if it's intended to stack on multiple Mirrored Technique+ usages, or if it's not, this will need custom stacking behavior to take the Math.max value on stack (to cover both orders of Mirrored Technique and Mirrored Technique+)